src/main/java/repsmax/Repsmax.java

```diff
@@ -0,0 +1,61 @@
+package repsmax;
```

Comment:
```suggestion
package repsmax; Neat coding style
```
Great job on writing such a neat code!
Really Impressed by the clarity enabling efficient code reading. 🎉